src/components/select/MultiSelectConnected.tsx

```diff
@@ -66,7 +66,7 @@ const parentDropTarget = {
 @DropTarget(DraggableSelectedOptionType, parentDropTarget, (connect: any) => ({
     connectDropTarget: connect.dropTarget(),
 }))
-export class MultiSelectConnected extends React.Component<IMultiSelectProps, {}> {
+export class MultiSelectConnected extends React.Component<IMultiSelectProps & React.ButtonHTMLAttributes<HTMLButtonElement>> {

```

Comment:
Nice call. to help with other project who want to use the IMultiSelectProps interface, we need to do this
`interface IMultiSelectProps extends React.ButtonHTMLAttributes<HTMLButtonElement>`

Reply:
changed, thanks